### ISSUE TITLE
Keboola\Csv\Exception: Cannot write resource into a column

### ISSUE THREAD

**Issue by ondrejhlavacek** [keboola/db-extractor-pgsql#55]
https://connection.keboola.com/admin/utils/logs?file=2019/01/29/10/2019-01-29-10-37-56-5c501ef4f2e59-exception

https://connection.keboola.com/admin/utils/logs?file=2019/01/29/10/2019-01-29-10-37-57-5c501ef513821-log

**Comment by pivnicek** [comment]
\copy command failed with `invalid byte sequence for encoding \"UTF8\": 0x92`

**Comment by pivnicek** [comment]
imho invalid UTF8 should be a user exception

**Comment by pivnicek** [comment]
Yes, because if the \copy command fails it tries via PDO,  the PDO failed with that CSV error, but I believe the root cause for both failures is the same

**Comment by tomasfejfar** [comment]
Happens again:
* https://keboola.slack.com/archives/C09U3R1J4/p1551651552000100
* https://keboola.slack.com/archives/C09U3R1J4/p1551653346000200
* https://keboola.slack.com/archives/C09U3R1J4/p1551682510001100

**Comment by kacurez** [comment]
https://keboola.slack.com/archives/C09U3R1J4/p1554855965020500

**Comment by Halama** [comment]
Again, it is really strange issue. 
https://keboola.slack.com/archives/C09U3R1J4/p1555971642020900

thrown here https://github.com/keboola/php-csv/blob/eb77f51e22d3e13a8e14a5d5a0b588cf9e8c56c5/src/CsvFile.php#L355
